static/images/icon-custom-outputs.svg

```diff
@@ -0,0 +1,36 @@
+<svg fill="#777" stroke="#fff" width="100pt" height="100pt" version="1.1" viewBox="0 0 100 100" xmlns="http://www.w3.org/2000/svg">
```

Comment:
Just noting that I'm impressed that you were able to find an icon that is substantially different form the icons while also accurately representing the concept. Nice!